contracts/Billing.sol

```diff
@@ -59,9 +66,11 @@ contract Billing is IBilling, Governed {
     constructor(
         address _gateway,
```

Comment:
This is unrelated to the scope of adding the token gateway (bridge) but I'd use the opportunity to rename `gateway` to `agent` or something different to avoid mistakes, the role is really a funds collection agent. In addition I think I'd make it so we can configure multiple agents.

Reply:
Good idea. How about `collector`? (and using a mapping of authorized collectors)